app/models/pipeline_run.rb

```diff
@@ -26,9 +28,11 @@ class PipelineRun < ApplicationRecord
   PIPELINE_VERSION_FILE = "pipeline_version.txt".freeze
   STATS_JSON_NAME = "stats.json".freeze
   ERCC_OUTPUT_NAME = 'reads_per_gene.star.tab'.freeze
+  AMR_OUTPUT_NAME = 'output__fullgenes__ARGannot_r2__results.txt'.freeze
```

Comment:
Since you define this constant here, you should reuse it in host_filter.json.erb, rather than typing it out again. That way, the consistency of the filename you need in pipeline_run.rb and the filename that gets generated in idseq-dag is always guaranteed. It's more maintainable if you only need to modify the filename in 1 constant, rather than 2 places of the codebase.

Reply:
for sure, I don't think the constant was recognized when I moved it to the json.erb? Did <%= @AMR_OUTPUT_NAME%> (now this constant is called some thing else, but still)

Reply:
You can access it from outside the class using `PipelineRun:: AMR_OUTPUT_NAME`

Reply:
Hmm I got a JSON format error when doing it that way -- will mark this as a TODO and figure out how to get it in there.